decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- [ ] in the `Decorator` class you should include the attribute accessor for `nameable`